src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -43,6 +43,12 @@ public static class PrereleaseCompatibilityTransformer
 
             switch (version)
             {
+                case "2.0.0-csd.2.beta.2019-02-20":
+                {
+                    // SARIF TC31. Nothing to do.
+                    break;
+                }
+
                 case "2.0.0-csd.2.beta.2019-01-24":
                 {
                     // SARIF TC31. Nothing to do.
```

Comment:
>Nothing to do [](start = 35, length = 13)

There _is_ something to do between TC 31 and TC 32: You have to move any `invocation.attachments` and `tool.extensions`. The properties between an `attachment` and a `toolComponent` (the array element of `tool.extensions`) don't completely line up, but they do at least both have file locations.

@mikefan@microsoft.com If you think nobody has used `invocation.attachments` and it's not worth doing this, that's fine, but it deserves at least a comment.